lib/ret_web/router.ex

```diff
@@ -127,9 +128,9 @@ defmodule RetWeb.Router do
       resources("/slack", Api.V1.SlackController, only: [:create])
     end
 
-    scope "/v1/internal", as: :api_v1 do
+    scope "/api-internal/v1", as: :api_v1 do
       pipe_through([:portal_header_auth])
-      get("/presence", Api.V1.PresenceController, :show)
+      get("/presence", ApiInternal.PresenceController, :show)
```

Comment:
Looks like you are missing V1 here?
```suggestion
      get("/presence", ApiInternal.V1.PresenceController, :show)
```